### ISSUE TITLE
gappa analyze graft runtime error

### ISSUE THREAD

**Issue by gavinmdouglas** [lczech/gappa#6]
Hi @lczech ,

I ran into the below error when running an unusual dataset with EPA-NG (v0.3.4) and GAPPA (v0.1.0).
 
(Incidentally I just noticed that the version reported by GAPPA is still 0.0.0)

You can download the input jplace file here: https://www.dropbox.com/s/qrx39hy8ahosmx8/epa_result.jplace?dl=1

The command and output:

```
gappa analyze graft --fully-resolve --jplace-path epa_result.jplace

v0.0.0, (c) 2017-2018 by Lucas Czech and Pierre Barbera

Found 1 jplace files.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Expected end of input while reading Json at 777682:1
Aborted
```

Although the placed sequences are unusual (they were "clustered" at 100% identity rather than a lower identity as is typically done for 16S sequences, and there are 83,354 in total) I was able to get an output newick tree with ```gappa tog``` with the same file and it appears to be in correct JSON format.

Any idea why this is occurring?

**Comment by lczech** [comment]
Hi @gavinmdouglas,

thanks for reporting the issue. The bug was introduced a few weeks ago while I refactored some code, and it seems I didn't notice so far... 

Anyway, it is fixed now, so please use the newly released [gappa v0.2.0](https://github.com/lczech/gappa/releases/tag/v0.2.0). It works now with the test file that you sent, so I'm closing the issue. If you still run into trouble, feel free to re-open.

Best
Lucas

**Comment by gavinmdouglas** [comment]
Great, thanks!
